mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/MimePart.java

```diff
@@ -218,22 +218,22 @@ public HeaderCollection getHeaderCollection() {
         return fileName;
     }
 
-    @JsonProperty(JsonMessageConstants.Attachment.FILE_EXTENSION)
+    @JsonIgnore
     public Optional<String> getFileExtension() {
         return fileExtension;
     }
 
-    @JsonProperty(JsonMessageConstants.Attachment.MEDIA_TYPE)
+    @JsonIgnore
     public Optional<String> getMediaType() {
         return mediaType;
     }
 
-    @JsonProperty(JsonMessageConstants.Attachment.SUBTYPE)
+    @JsonIgnore
     public Optional<String> getSubType() {
         return subType;
     }
```

Comment:
I don't agree for these 3 @Ignore

Reply:
It's not @Ignore but JsonIgnore.

Well it may come from the fact MimePart is both the Json model and the piece of logic flattening a message, and this should be addressed yes.

I'll add a commit.

Reply:
That was not my worry. The fact that no query use this field doesn't mean we should drop them.
It's important to be able to search attachment by their types.
Do we have others solutions to reach this goal?